### PR TITLE
edits to headings for parallelism

### PR DESCRIPTION
**Pull request by jantley-ibm**

### CODE REVIEW

**Comment by jantley-ibm** [comment]
@dmuelle should be a quick review, thanks

File: pipelines.adoc [hunk 1]
```diff
@@ -42,9 +42,9 @@ When the Kabanero operator activates the CRD, it associates the pipelines in the
 
 === Updating default tasks and pipelines
 
-The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive for use by the Kabanero CRD is to run the  link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh[package.sh] script or run the travis build against a release of your pipelines repo.  The script generates the archive file with the necessary pipeline artifacts and a manifest.yaml that describes the contents of the archive.  
+The default tasks and pipelines can be updated by forking the Kabanero Pipelines repo and editing the files under `pipelines/incubator`.  The easiest way to generate the archive for use by the Kabanero CRD is to run the  link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh[package.sh] script or run the travis build against a release of your pipelines repo.  The script generates the archive file with the necessary pipeline artifacts and a manifest.yaml that describes the contents of the archive.
```
Comment (dmuelle) [inline pipelines.adoc]: Does running the Travis build also generate an archive file? It's mentioned but not explained.  Might be better to break off into a separate sentence. Also manifest.yaml needs monospace and a noun descriptor ("file").

The easiest way to generate the archive for use by the Kabanero CRD is to run the  link:https://github.com/kabanero-io/kabanero-pipelines/blob/master/ci/package.sh[package.sh] script.  The script generates the archive file with the necessary pipeline artifacts and a `manifest.yaml` file that describes the contents of the archive. Alternatively, you can run the travis build against a release of your pipelines repo, which also generates an archive file and a `manifest.yaml` file.

..or whatever explanation applies to the travis build option.

**Review by dmuelle — COMMENTED** [review]
headings look good- question in the comments re updating defaults section

File: pipelines.adoc (hunk 1 above)
Comment (jantley-ibm) [reply pipelines.adoc]: @kvijai82 could you comment on @dmuelle 's question?  I'll make any edits if needed -- thanks

Comment (kvijai82) [reply pipelines.adoc]: @dmuelle yes the travis build basically runs the same package.sh script and will generate the same manifest file and archive file.  I would like to work on a short guide that breaks this down in more detail this week.

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**